BackEnd/src/controllers/DeleteProject.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+namespace HostMyDocs\Controllers;
+
+use Chumper\Zipper\Zipper;
+use Slim\Container;
+use Slim\Http\Response as Response;
+use Slim\Http\Request as Request;
+use Slim\Http\UploadedFile;
+use Symfony\Component\Filesystem\Filesystem;
+
+class DeleteProject extends BaseController
+{
+    /**
+     * @var null|string Name of the project to add
+     */
+    private $name = null;
+
+    /**
+     * @var null|string SemVer representation of the version of the project to add
+     */
+    private $version = null;
+
+    /**
+     * @var null|string Programming language of the project to add
+     */
+    private $language = null;
+
+    /**
+     * @var null|Filesystem Symfony Filesystem wrapper
+     */
+    private $filesystem = null;
+
+    public function __construct(Container $container)
+    {
+        parent::__construct($container);
+        $this->filesystem = new Filesystem();
+    }
+
+    /**
+     * Main method of the controller
+     *
+     * @param Request $request
+     * @param Response $response
+     * @return Response
+     */
+    public function __invoke(Request $request, Response $response)
+    {
+        // increasing execution time
+        ini_set('max_execution_time', 3600);
```

Comment:
Is it really necessary to override `max_execution_time` ? (default is 30 seconds)